.drone.yml

```diff
@@ -44,14 +44,14 @@ pipeline:
       COMMIT: ${DRONE_COMMIT}
     commands:
       - make mark
-      - make -j3 all
+      - make all
       - make sincemark
       - make mark
       - echo `ls vendor/github.com/vmware/govmomi/vim25/methods`
       - echo `ls vendor/github.com/vmware/govmomi/vim25/types`
       - echo `ls vendor/github.com/docker/docker/vendor/github.com/opencontainers/runc/libcontainer/system`
       - export VIC_ESX_URL_ARRAY="`tests/get_test_url.sh`"
-      - make -j3 test
+      - make test
```

Comment:
I think for the `go test` portion, I would like make -j3 since I believe go will only parallelize for the compile step. @hickeng @caglar10ur can clarify if this is true.